Sources/VMwareFusion/VMwareFusion.swift

```diff
@@ -0,0 +1,112 @@
+//
+//  VMwareFusion.swift
+//  gitlab-fusion
+//
+//  Created by Ryan Lovelett on 10/9/20.
+//
+
+import Foundation
+import os.log
+import Path
+
+/// A type that manages interactions with VMware Fusion. Specifically it
+/// interacts with `vmrun` to be able to manage VMware Fusion guests.
+struct VMwareFusion<Subprocess: Executable> {
```

Comment:
Can I move this generic to the init so that it does not pollute all the other call sites?